### PR TITLE
encourages interdepartmental cooperation

### PR DESCRIPTION
**Pull request by Hatterhat**
## About The Pull Request
moves m41a small AP and HP magazines to the protolathe, adds small M41A magazines to the protolathe
as a consolation prize the pf gets snarky magazines in their locker
## Why It's Good For The Game
the autolathe, historically, has only spit out default and hunter ammunition
so has the protolathe, but nobody wants to have that conversation
## Changelog
:cl:
balance: Small M41A hollow-point and armor-piercing magazines have been moved to the protolathe.
/:cl:

### CODE REVIEW

File: code/modules/projectiles/guns/projectile/caseless.dm [hunk 1]
```diff
@@ -1,6 +1,6 @@
 /obj/item/gun/projectile/caseless/prototype
 	name = "prototype caseless rifle"
-	desc = "A rifle cooked up in NanoTrasen's R&D labs that operates with Kraut Space Magic� clockwork internals. Uses solid phoron 5mm caseless rounds."
+	desc = "A rifle cooked up in NanoTrasen's R&D labs that operates with Kraut Space Magic� clockwork internals. Uses solid phoron 5mm caseless rounds."
```
Comment (lolman360) [inline code/modules/projectiles/guns/projectile/caseless.dm:3]: not again

Comment (KasparoVy) [reply code/modules/projectiles/guns/projectile/caseless.dm:3]: YES!!! THE <?>

Comment (Hatterhat) [reply code/modules/projectiles/guns/projectile/caseless.dm:3]: god damn it

**Review by silicons — APPROVED** [review]
i'm honestly not sure why anyone put special ammo types in the generic autolathe instead of protolathe or some kind of special cargo order
what's the point of special ammo types if you can just print it effortlessly with metal and glass

**Pull request merged**